### PR TITLE
refactor: hook for change requests

### PR DESCRIPTION
**Pull request by kwasniew**
## About the changes

Centralized change requests enable/disable in one hook.
Rationale: we can swap it to proper settings instead of a feature flag at some point.

### CODE REVIEW

File: frontend/src/component/project/Project/ProjectFeatureToggles/ProjectFeatureToggles.tsx [hunk 1]
```diff
@@ -38,6 +38,7 @@ import { useMediaQuery } from '@mui/material';
 import { Search } from 'component/common/Search/Search';
 import { useChangeRequestToggle } from 'hooks/useChangeRequestToggle';
 import { ChangeRequestDialogue } from 'component/changeRequest/ChangeRequestConfirmDialog/ChangeRequestConfirmDialog';
+import { useChangeRequestsEnabled } from '../../../../hooks/useChangeRequestsEnabled';
```
Comment (Tymek) [inline frontend/src/component/project/Project/ProjectFeatureToggles/ProjectFeatureToggles.tsx:41]:
```suggestion
import { useChangeRequestsEnabled } from 'hooks/useChangeRequestsEnabled';
```
should work too

File: frontend/src/component/project/Project/Project.tsx [hunk 2]
```diff
@@ -28,6 +28,7 @@ import { ChangeRequestOverview } from 'component/changeRequest/ChangeRequestOver
 import { DraftBanner } from 'component/changeRequest/DraftBanner/DraftBanner';
 import { MainLayout } from 'component/layout/MainLayout/MainLayout';
 import { ProjectChangeRequests } from '../../changeRequest/ProjectChangeRequests/ProjectChangeRequests';
+import { useChangeRequestsEnabled } from '../../../hooks/useChangeRequestsEnabled';
```
Comment (Tymek) [inline frontend/src/component/project/Project/Project.tsx:31]:
```suggestion
import { useChangeRequestsEnabled } from 'hooks/useChangeRequestsEnabled';
```

File: frontend/src/component/feature/FeatureStrategy/FeatureStrategyRemove/FeatureStrategyRemove.tsx [hunk 3]
```diff
@@ -15,6 +15,7 @@ import PermissionIconButton from 'component/common/PermissionIconButton/Permissi
 import { Delete } from '@mui/icons-material';
 import useUiConfig from '../../../../hooks/api/getters/useUiConfig/useUiConfig';
 import { useChangeRequestApi } from '../../../../hooks/api/actions/useChangeRequestApi/useChangeRequestApi';
+import { useChangeRequestsEnabled } from '../../../../hooks/useChangeRequestsEnabled';
```
Comment (Tymek) [inline frontend/src/component/feature/FeatureStrategy/FeatureStrategyRemove/FeatureStrategyRemove.tsx:18]:
```suggestion
import { useChangeRequestsEnabled } from 'hooks/useChangeRequestsEnabled';
```

File: frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvSwitches/FeatureOverviewEnvSwitch/FeatureOverviewEnvSwitch.tsx [hunk 4]
```diff
@@ -13,6 +13,7 @@ import { useRequiredPathParam } from 'hooks/useRequiredPathParam';
 import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
 import { useChangeRequestToggle } from 'hooks/useChangeRequestToggle';
 import { ChangeRequestDialogue } from 'component/changeRequest/ChangeRequestConfirmDialog/ChangeRequestConfirmDialog';
+import { useChangeRequestsEnabled } from '../../../../../../hooks/useChangeRequestsEnabled';
```
Comment (Tymek) [inline frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvSwitches/FeatureOverviewEnvSwitch/FeatureOverviewEnvSwitch.tsx:16]:
```suggestion
import { useChangeRequestsEnabled } from 'hooks/useChangeRequestsEnabled';
```

**Review by Tymek — COMMENTED** [review]
I think we should use absolute imports here. I know it's easier to do relative because editor autocomplete does it this way

**Comment by kwasniew** [comment]
@Tymek good catch. TBH I didn't even look at the imports as the IDE took care of it :)

**Review by Tymek — APPROVED** [review]

**Pull request merged**